app/schemas/models/campaign.schema.coffee

```diff
@@ -78,6 +78,20 @@ _.extend CampaignSchema.properties, {
       #- normal properties
       position: c.point2d()
 
+      # properties relevant for voyager campaigns
+      nextLevels: {   # array of next levels
+        type: 'array'
+        format: 'next-level'
```

Comment:
What does this format field do?

Reply:
Good catch, I think I tried to use it while debugging an issue but it didnt help (not sure what it does entirely). Will remove. Thanks.